R/mlr_reflections.R

```diff
@@ -125,7 +125,7 @@ local({
   )
 
   ### Learner
-  tmp = c("featureless", "missings", "weights", "importance", "selected_features", "oob_error", "loglik", "hotstart_forward", "hotstart_backward")
+  tmp = c("featureless", "missings", "weights", "importance", "selected_features", "oob_error", "loglik", "hotstart_forward", "hotstart_backward", "marshal")
```

Comment:
do we really need this?

Reply:
--> we either keep the property or the default marshal method

Reply:
Even if we identify this I think it is nice to have this in the overview of the learner properties